subsys/net/l2/wifi/wifi_mgmt.c

```diff
@@ -26,10 +26,10 @@ static int wifi_connect(u32_t mgmt_request, struct net_if *iface,
 		return -ENOTSUP;
 	}
 
-	NET_DBG("%s %u %u %u %s %u",
-		params->ssid, params->ssid_length,
-		params->channel, params->security,
-		params->psk, params->psk_length);
+	LOG_HEXDUMP_DBG(params->ssid, params->ssid_length, "ssid");
+	LOG_HEXDUMP_DBG(params->psk, params->psk_length, "psk");
+	NET_DBG("%u %u",
```

Comment:
Looking the print again, I think we could add here a bit more information about what the two numbers are. So something like `ch %u sec %u`